patches/server/0845-Add-player-health-update-API.patch

```diff
@@ -0,0 +1,18 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SamB440 <sam@islandearth.net>
+Date: Wed, 17 Nov 2021 12:31:42 +0000
+Subject: [PATCH] Add player health update API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 4ff4143f3a7cd89ef92f4b8882fa3e5addfe0f06..01f18c29a4107a11b3e6d607658868f302f557d7 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -2009,6 +2009,7 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+         this.getHandle().maxHealthCache = getMaxHealth();
+     }
+ 
++    @Override // Paper
```

Comment:
Comment for clarity's sake; the server patch is not needed, but it's fine to stay as a forced diff on method move/change